Requesting Pull

djobes opened the pull request:
I added the Plex Test app and a hand full of button keys from the remote.
Up, Down, Left, Right, Back, Select.

djobes committed:
Added Roku Remote buttons functions: Up,Down,Left,Right,Back, and Select.

Added remote button functions: Up,Down,Left,Right,Back,Select,Forward.
Example: Alexa ask Roku to go "Left"

Also added PlexTest app open function (#14987).  This is the Plex app old format version.
So me people like this format better.

djobes committed:
Fix duplication of all code in server file.

julianh2o commented:
Thanks for the contributions! I've added you to the list of contributors and merged your changes! (I made a few very minor tweaks that you may want to look out for if you merge this back into your fork)

djobes commented:
Thanks So i should merge back to correct my fork?Will do. 


David Jobes-------- Original message --------From: Julian Hartline <notifications@github.com> Date: 12/23/2016  8:48 PM  (GMT-06:00) To: julianh2o/RokuAlexaLambdaSkill <RokuAlexaLambdaSkill@noreply.github.com> Cc: djobes <djobes@jobes.com>, Author <author@noreply.github.com> Subject: Re: [julianh2o/RokuAlexaLambdaSkill] Requesting Pull (#8) 
Thanks for the contributions! I've added you to the list of contributors and merged your changes! (I made a few very minor tweaks that you may want to look out for if you merge this back into your fork)

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or mute the thread.


  
  




{"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/julianh2o/RokuAlexaLambdaSkill","title":"julianh2o/RokuAlexaLambdaSkill","subtitle":"GitHub repository","main_image_url":"https://cloud.githubusercontent.com/assets/143418/17495839/a5054eac-5d88-11e6-95fc-7290892c7bb5.png","avatar_image_url":"https://cloud.githubusercontent.com/assets/143418/15842166/7c72db34-2c0b-11e6-9aed-b52498112777.png","action":{"name":"Open in GitHub","url":"https://github.com/julianh2o/RokuAlexaLambdaSkill"}},"updates":{"snippets":[{"icon":"PERSON","message":"@julianh2o in #8: Thanks for the contributions! I've added you to the list of contributors and merged your changes! (I made a few very minor tweaks that you may want to look out for if you merge this back into your fork)"}],"action":{"name":"View Pull Request","url":"https://github.com/julianh2o/RokuAlexaLambdaSkill/pull/8#issuecomment-269065505"}}}
